<launch>
    <rosparam subst_value="true">
        touch:
            head_threshold: 100
            chest_threshold: 150
        pan_tilt:
            pan_id: 3
            pan_p: 65
            pan_d: 50
            tilt_id: 4
            tilt_p: 95
            tilt_d: 20
            min_pan_angle: -50
            max_pan_angle: 50
            min_tilt_angle: -15
            max_tilt_angle: 15
            min_playtime: 50
            max_playtime: 200
        http_server:
            address: localhost
            port: 8000
            static: $(find elmo)/src/static
        wifi:
            credentials: /home/idmind/wifi.json
        gpio:
            shutdown_pin: 17
    </rosparam>
    <node pkg="elmo" type="pan_tilt.py"         name="pan_tilt" />
    <node pkg="elmo" type="gpio.py"             name="gpio" />
    <node pkg="elmo" type="leds.py"             name="leds" />
    <node pkg="elmo" type="touch_sensors.py"    name="touch_sensors" />
    <node pkg="elmo" type="http_server.py"      name="http_server" />
    <node pkg="elmo" output="screen" type="robot_server.py"     name="robot_server" />
    <node pkg="elmo" output="screen" type="wifi.py"             name="wifi" />
    <!-- test nodes -->
    <node pkg="elmo" output="screen" type="test_motors.py"      name="test_pan_tilt" />
    <node pkg="elmo" output="screen" type="test_leds.py"        name="test_leds" />
    <!-- startup routine -->
    <!--node pkg="elmo" output="screen" type="start_routine.py"    name="start_routine" /-->
    <!--node pkg="elmo" output="screen" type="start_demo.py"    name="start_demo" /-->

</launch>
